aws/resource_aws_cloudfront_field_level_encryption_profile.go

```diff
@@ -0,0 +1,208 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAwsCloudfrontFieldLevelEncryptionProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCloudfrontFieldLevelEncryptionProfileCreate,
+		Read:   resourceAwsCloudfrontFieldLevelEncryptionProfileRead,
+		Update: resourceAwsCloudfrontFieldLevelEncryptionProfileUpdate,
+		Delete: resourceAwsCloudfrontFieldLevelEncryptionProfileDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"encryption_entities": {
```

Comment:
Is order significant here? i.e. prefer `schema.TypeSet` over `schema.TypeList`?

Reply:
changed

Reply:
I Changed it Set and added a test (locally) but i see that it doesnt allow more that single one so order doesnt matter. it does look cleaner to use `schema.Set` instead of `[]interface{}` 
ill add a MaxItems:1 to it.